cluster-autoscaler/clusterstate/utils/types.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
I meant:

* LogItem types to ./clusterstate/api. It may eventually go to the end user some day via an api.
* LogCollector code ./clusterstate/utils (or ./utils). It will never get to the user.

Maybe I wasn't clear enough. Sorry for confusion.